tt_metal/impl/kernels/kernel.cpp

```diff
@@ -376,7 +376,7 @@ void ComputeKernel::read_binaries(Device *device) {
     std::vector<ll_api::memory> binaries;
     for (int trisc_id = 0; trisc_id <= 2; trisc_id++) {
         const JitBuildState &build_state = device->build_kernel_state(JitBuildProcessorType::COMPUTE, trisc_id);
-        ll_api::memory binary_mem = llrt::get_risc_binary(build_state.get_target_out_path(this->kernel_full_name_));
+        ll_api::memory binary_mem = llrt::get_risc_binary(build_state.get_target_out_path(this->kernel_full_name_), trisc_id + 2, llrt::PackSpans::PACK);
```

Comment:
NIT: Should these hardcoded offsets be defines? Something like `TRISC_ID_BASE` and `ERISC_ID_BASE`.